cs-rin-ru-enhanced-mod.user.js

```diff
@@ -1263,6 +1412,118 @@ async function specialSearch() {
 
 specialSearch();
 
+/*
+Originally made by ucsanytaef
+And adapted for cs.rin.ru enhanced by Altansar (nothing to adapt xD)
+*/
+function ShowAllSpoilers() {
+    if (options.show_all_spoilers) { //If show all spoilers is active
+        setTimeout(function () {
+            var spoilers = document.querySelectorAll('input[type="button"][value="Show"]');
+            if (spoilers.length > 0) {
```

Comment:
This is not needed, if the list is empty, for loop will just do nothing

Reply:
Yeah idk why i write this.